models/issues/issue.go

```diff
@@ -934,3 +937,33 @@ func insertIssue(ctx context.Context, issue *Issue) error {
 
 	return nil
 }
+
+// ChangeIssueTimeEstimate changes the plan time of this issue, as the given user.
+func ChangeIssueTimeEstimate(issue *Issue, doer *user_model.User, timeEstimate int64) (err error) {
```

Comment:
can you inject the context here

Reply:
https://github.com/go-gitea/gitea/pull/23113/commits/823d67f27892bf744f257a098709745f38633e6d